### PR TITLE
add reusability for calling functions with cron on FluidProposals contract

### PR DESCRIPTION
**Pull request by ArmmGh**
Added dynamic route for fluid-proposals contract to reuse the same logic and support adding new crons and functions easily

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -2,8 +2,12 @@
   "buildCommand": "pnpm turbo build",
   "crons": [
     {
-      "path": "/api/cron/fluid-proposal",
+      "path": "/api/cron/fluid-proposal/sync",
       "schedule": "0 * * * *"
+    },
+    {
+      "path": "/api/cron/fluid-proposal/syncSupertoken",
+      "schedule": "* * * * *"
```
Comment (kamikazebr) [inline vercel.json]: It's will run each minute maybe we want it to run each 1 hour, the suggestion below its how it works.

```suggestion
      "schedule": "0 * * * *"
```

In fact maybe we want even more time between, maybe days. @0xGabi can tell us what could be ideal?

Example: each 2 days
```suggestion
      "schedule": "0 0 */2 * *"
```

Comment (ArmmGh) [reply vercel.json]: Sure, I should've been asked before, let's see what @0xGabi will suggest

Comment (kanaagawa) [reply vercel.json]: Hey, I think we can update the sync method to be 6 hours and the syncSupertoken depends on which parameters we end using, nevertheless something like 2-5 days sounds a good first estimation

**Comment by ArmmGh** [comment]
Updated: 
1: sync called every 6 hour
1: syncSuperToken called every 3 days

**Review by kamikazebr — APPROVED** [review]
LGTM!

**Comment by ArmmGh** [comment]
Shall we merge this? @0xGabi @kamikazebr

**Comment by kamikazebr** [comment]
image
Sorry for delay @ArmmGh

**Pull request merged**